ThunderCloud/QuizQuestionContainerViewController.swift

```diff
@@ -148,8 +148,11 @@ open class QuizQuestionContainerViewController: AccessibilityRefreshingViewContr
         
         super.viewDidLoad()
 
-        navigationItem.rightBarButtonItem = UIBarButtonItem(image: (#imageLiteral(resourceName: "quiz-dismiss") as StormImageLiteral).image, style: .plain, target: self, action: #selector(handleQuitQuiz(_:)))
-        navigationItem.rightBarButtonItem?.accessibilityLabel = "Quit Quiz".localised(with: "_QUIZ_BUTTON_QUIT")
+        // If we're being presented and we're not the first view controller
```

Comment:
Comment says `&&`, logic says `||`. Is that intentional? :)

Reply:
You are correct of course!